py/orbit/parsers/madx_parser.py

```diff
@@ -718,15 +735,15 @@ def parseElem(self,line_init):
 		tokens = line_init.split(",")
 		subtokens = tokens[0].split(":")
 		name = subtokens[0]
-		type = subtokens[1]
+		type_local = subtokens[1]
 
 		# elem can be defined as a child node
 		paramsDict = {}
-		if type not in self._accElemDict.keys():			
-			type_upd = type
+		if type_local not in self._accElemDict.keys():			
+			type_upd = type_local
 		else:	
-			type_upd = self._accElemDict[type].getType()
-			paramsDict = self._accElemDict[type].getParameters()
+			type_upd = self._accElemDict[type_local].getType()
+			paramsDict = self._accElemDict[type_local].getParameters()
 
 		if "marker" in type_upd.lower():
 			type_upd = "marker"
```

Comment:
Dear Dimitrii,
you for got to remove 
if "monitor" in type_upd.lower():
			type_upd = "monitor"
The issue with the monitors have been solve and the solution is commit  in the current master branch

Reply:
Ii solved. Misstake by me.